cli/borges/command.go

```diff
@@ -18,3 +21,23 @@ type queueOpts struct {
 	Queue  string `long:"queue" env:"BORGES_QUEUE" default:"borges" description:"queue name"`
 	Broker string `long:"broker" env:"BORGES_BROKER" default:"amqp://localhost:5672" description:"broker service URI"`
 }
+
+type metricsOpts struct {
+	Metrics     bool `long:"metrics" env:"BORGES_METRICS" description:"expose a metrics endpoint using an HTTP server"`
+	MetricsPort int  `long:"metrics-port" env:"BORGES_METRICS_PORT" description:"port to bind metrics to" default:"6062"`
```

Comment:
I would make it `BORGES_METRICS_ADDR` with default `:6062`
if you want to use 0.0.0.0 or 127.0.0.1 or even debug and send it out to the external server you'll be able.